lib/Sema/TypeCheckAccess.cpp

```diff
@@ -969,69 +972,89 @@ void UsableFromInlineChecker::check(Decl *D) {
     auto PBD = cast<PatternBindingDecl>(D);
     bool isTypeContext = PBD->getDeclContext()->isTypeContext();
 
+    // Stored instance properties in public/@usableFromInline fixed-contents
```

Comment:
I know it was already like this when you got here but the PBD case here is too long. Do you mind splitting it out into smaller methods, at least so that each can have a descriptive name and/or comment?

Reply:
I did a whole bunch of cleanups, to the point where I think they deserve a separate PR. I promise they're coming, though. :-)